exporter/azuremonitorexporter/README.md

```diff
@@ -1,6 +1,10 @@
 # Azure Monitor Exporter
 
-This exporter sends logs and trace data to [Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/).
```

Comment:
This line should be moved under the new status header

Reply:
done